lib/arturo/feature.rb

```diff
@@ -81,8 +81,7 @@ def self.last_updated_at
 
     protected
 
-    def passes_threshold?(feature_recipient)
-      threshold = self.deployment_percentage || 0
+    def passes_threshold?(feature_recipient, threshold)
```

Comment:
... at least docs would be nice ... `# used by external library, do not modify interface`
also means we are using a protected method ... making it public might be more obvious ... but either way is fine